src/test/java/cucumber/steps/Hooks.java

```diff
@@ -0,0 +1,33 @@
+package cucumber.steps;
+
+import api.ApiClient;
+import exeptions.FailedToLoginException;
+import helpers.UserLogin;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.BeforeAll;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class Hooks {
+    private static final Logger LOGGER = LogManager.getLogger();
+    private static ApiClient api = new ApiClient();
```

Comment:
You have constuctor with ApiClient argument in many classes, and send there newly created instance of ApiClient each time.
In this case, what is the use to keep it in the constructor?

Reply:
Don't understand the question. In Hooks ApiClient is for login only.